app/views/subscriptions/feedback_received.html.haml

```diff
@@ -0,0 +1,12 @@
+- content_for :page_title_prefix, t('.title')
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    .govuk-panel.tvs-panel.govuk-panel--confirmation
```

Comment:
the class tvs-panel isnt needed. i will do it as another small PR myself but the whole usage of tvs-panel should be removed.

Reply:
If we use the `GovukComponent::Panel`, there is a PR for that already, we can clean it there

Reply:
actually we need to catch up about this in the morning, just need to align on a few things as this has become linked in an odd way to my current PR and your other one for the gov-uk panel